autogpt_platform/frontend/src/components/nav/NavBarButtons.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+import { BsBoxes } from "react-icons/bs";
+import { LuLaptop, LuShoppingCart } from "react-icons/lu";
+import { cn } from "@/lib/utils";
+import { usePathname } from "next/navigation";
+import { getBehaveAs } from "@/lib/utils";
+import MarketPopup from "./MarketPopup";
+
+export function NavBarButtons({ className }: { className?: string }) {
```

Comment:
If you moved this, you need to move CreditButton too

Reply:
Done